src/training/scheduler.h

```diff
@@ -169,8 +169,11 @@ class Scheduler : public TrainingObserver {
 
     // stop if the first validator did not improve for a given number of checks
     size_t stopAfterStalled = options_->get<size_t>("early-stopping");
-    if(stopAfterStalled > 0 && !validators_.empty()
-       && stalled() >= stopAfterStalled)
+    if(stopAfterStalled > 0 && !validators_.empty() && stalled() >= stopAfterStalled)
```

Comment:
This change seems unrelated. Can it be done in a different PR? Or please respond to this comment  to explain what this is about

Reply:
Reverted. Such kind of changes are usually a mix of the boy scout rule and hitting the autoformat shortkey too frequently. I agree that changes should be kept minimal.